### PR TITLE
DR-1593 - part 2: Grab tag from bumper instead of separate action step

### PR DESCRIPTION
**Pull request by snf2ye**
Successful run here: https://github.com/DataBiosphere/jade-data-repo/actions/runs/537604980

### CODE REVIEW

File: .github/workflows/dev-image-update.yaml [hunk 1]
```diff
@@ -54,44 +55,73 @@ jobs:
...
         run: |
-          git checkout ${{ steps.apiprevioustag.outputs.tag }}
+          git checkout ${{ steps.bumperstep.outputs.tag }}
       - name: 'Release Candidate Container Build: Checkout DataBiosphere/jade-data-repo-ui repo'
         uses: actions/checkout@v2
         with:
           repository: 'DataBiosphere/jade-data-repo-ui'
           token: ${{ secrets.HELM_REPO_TOKEN }}
           path: jade-data-repo-ui
           ref: develop
```
Comment (snf2ye) [inline .github/workflows/dev-image-update.yaml:77]: I actually don't know why we need this step. This image update is just for API, not UI.

Comment (snf2ye) [reply .github/workflows/dev-image-update.yaml:77]: (I could try to test this out in a separate PR)

File: build.gradle [hunk 2]
```diff
@@ -43,7 +43,7 @@ plugins {
 
 allprojects {
     group 'bio.terra'
-    version '1.6.0-SNAPSHOT'
+    version '1.9.0-SNAPSHOT'
```
Comment (nmalfroy) [inline build.gradle:46]: Doesn't this get automatically bumped?

**Review by nmalfroy — APPROVED** [review]
much cleaner, thanks!

File: build.gradle (hunk 2 above)
Comment (snf2ye) [reply build.gradle:46]: Yes, but while I'm testing it bumps it on my branch.

**Review by samanehsan — APPROVED** [review]
LGTM! :octocat:

**Pull request merged**